src/zarr_benchmarks/read_write_zarr_v3.py

```diff
@@ -16,14 +17,14 @@ def get_compression_ratio(store_path: pathlib.Path) -> float:
     return compression_ratio
 
 
-def read_zarr_array(store_path: pathlib.Path) -> np.array:
+def read_zarr_array(store_path: pathlib.Path) -> npt.NDArray:
     zarr_read = zarr.open_array(store_path, mode="r")
     read_image = zarr_read[:]
-    return read_image
+    return np.asarray(read_image)
```

Comment:
```suggestion
    return read_image
```
Same here - not sure you need `np.asarray`?